.github/workflows/build-and-publish.yaml

```
@@ -118,15 +107,91 @@ jobs:
+  create-manifest:
+    name: Create multi-arch manifest
+    needs: [build-amd64-ppc64le-s390x, build-arm64]
+    runs-on: ubuntu-latest
+    env:
+      BRANCH_PREFIX: redhat-
+      REGISTRY: ${{ secrets.REGISTRY || 'quay.io/projectquay' }}
+      REPO_NAME: ${{ github.event.repository.name }}
+      TAG_SUFFIX: -unstable
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.10)

[error] the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)